aea/manager/manager.py

```diff
@@ -700,15 +798,18 @@ def start_agent(self, agent_name: str) -> "MultiAgentManager":
         if self._is_agent_running(agent_name):
             raise ValueError(f"{agent_name} is already started!")
 
-        aea = agent_alias.get_aea_instance()
-
         if self._mode == "async":
-            task = AgentRunAsyncTask(aea, self._loop)
+            task = AgentRunAsyncTask(agent_alias, self._loop)
         elif self._mode == "threaded":
-            task = AgentRunThreadTask(aea, self._loop)
+            task = AgentRunThreadTask(agent_alias, self._loop)
+        elif self._mode == "multiprocess":
+            task = AgentRunProcessTask(agent_alias, self._loop)
 
         task.start()
+        time.sleep(1)
```

Comment:
why is this needed?

Reply:
removed, was added during investigating some issues